Sources/XcodeGenKit/FileWriter.swift

```diff
@@ -16,12 +16,11 @@ public class FileWriter {
         let tempPath = try Path.processUniqueTemporary() + "XcodeGen"
         try? tempPath.delete()
         if projectPath.exists {
-            try projectPath.copy(tempPath)
+            try projectPath.move(tempPath)
```

Comment:
If the project is open, this would yank the project file out from underneath Xcode, which would complain no?

Reply:
Thank you for your concern! I've tested the changes multiple times, as I mentioned in the issue description, both by using the scheme preAction and from the terminal. It appears that there are no issues with the project file in my environment, and I haven't encountered any complains in Xcode. If you have specific test cases or scenarios in mind that you'd like me to investigate, please feel free to share them!